app/src/main/java/org/wikipedia/views/PageActionOverflowView.kt

```diff
@@ -102,13 +102,11 @@ class PageActionOverflowView(context: Context) : FrameLayout(context), DonorBadg
     }
 
     private fun loadDonorInformation() {
-        if (ContributionsDashboardHelper.contributionsDashboardEnabled && AccountUtil.isLoggedIn.not()) {
-            binding.donorContainer.isVisible = false
-            return
+        if (ContributionsDashboardHelper.contributionsDashboardEnabled && AccountUtil.isLoggedIn) {
```

Comment:
Hi @Williamrai, I updated this logic a little bit. The previous logic will show the username item if `contributionsDashboardEnabled = false`, which is not correct.

Reply:
okay, thank you for finding this.